concepts/05 UI Components/Chat/20 Popup Integration.md

```diff
@@ -0,0 +1,273 @@
+One popular way to add a chat assistant to your website is to implement it in the modal at the bottom right corner of the screen. For this scenario, follow the steps below:
+
+- Place DevExtreme [Floating Action Button](/Documentation/Guide/UI_Components/Floating_Action_Button/Getting_Started_with_Floating_Action_Button/) at the bottom right of the window. Users can click it to open the popup.
+- Position our [Popup](/Documentation/Guide/UI_Components/Popup/Overview/) at the bottom right (or another preferred location).
+- Add the Chat component for the virtual assistant.
+
+The following code snippet demonstrates this scenario. Check the `CSS` tab for instructions on how to integrate Popup and Chat seamlessly.
+
+---
+##### jQuery
+
+    <!-- tab: index.html -->
+    <div id="action"></div>
+    <div id="popup"></div>
+
+    <!-- tab: index.js -->
+    const assistant = {
+        id: "02",
+        name: "Assistant"
+    };
+
+    const messages = [{
+        id: 1,
+        timestamp: new Date(),
+        author: assistant,
+        text: "Hello! How can I help you?"
+    }];
+
+    $(() => {
+        $("#action").dxSpeedDialAction({
+            label: "Chat",
+            icon: "send",
+            onClick() {
+                popup.show();
+            }
+        }).dxSpeedDialAction("instance");
+        const popup = $("#popup").dxPopup({
+            position: "right bottom",
+            width: 400,
+            height: 650,
+            shading: false,
+            contentTemplate: function (contentElement) {
+                contentElement.append(
+                $("<div />")
+                    .attr("id", "chat")
+                    .dxChat({
+                        width: 400,
+                        height: 600,
+                        onMessageEntered: (e) => {
+                            e.component.renderMessage(e.message);
+                        },
+                        items: messages
+                    })
+                );
+            }
+        }).dxPopup("instance");
+    });
+
+    <!-- tab: index.css -->
+    #chat {
+        border: none;
+    }
+
+    .dx-popup-content-scrollable {
+        padding: 0;
+        overflow: hidden;
+    }
```

Comment:
The closing button is indented too far to the right, no? Let's discuss it in the post.

.dx-popup-title.dx-toolbar { padding: 0 20px; }